ui/View.elm

```diff
@@ -314,7 +325,7 @@ viewSourceDetails source =
                 Just revision ->
                     let
                         mozillaHgUrl =
-                            treeToMozillaHgUrl source.tree
+                            treeToMozillaHgUrl <| Maybe.withDefault "" source.tree
```

Comment:
Would it make sense to change the signature of `treeToMozillaHgUrl` to take a `Maybe String` instead?